src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let results = days * 40;
+
+  if (days >= 7) {
+    results = results - 50;
+  } else if (days >= 3) {
+    results = results - 20;
+  }
```

Comment:
Consider simplifying the `if else` structure by using `if` with `return` to make the code easier to follow.